combat/test_unit.py

```diff
@@ -83,7 +84,8 @@ def test_means():
 precision = None
 mod = []
 dat = matrix.values
-batchmod = define_batchmod(batch)
```

Comment:
@guillaumeap Is the `define_batchmod` function useless now?

Reply:
I think it is, the function `model_matrix` too